app/serialization.py

```diff
@@ -241,6 +241,10 @@ def _get_unculled_hosts(group, org_id):
     return hosts
 
 
+def serialize_incomplete_group(group_id, org_id):
+    return {"id": _serialize_uuid(group_id), "org_id": org_id, "status": "creation_in_progress"}
```

Comment:
Just a nitpick, not a big deal and might even not be the best place to define the status and message.
```suggestion
    return {"id": _serialize_uuid(group_id), "org_id": org_id, "status": "accepted", "message": "The requested group is being created asynchronously."}
```

Happy to not dive in this rabbit hole and just ignore for now :)

Reply:
Thanks @ezr-ondrej !